_rules/7d6734.md

```diff
@@ -0,0 +1,207 @@
+---
+id: 7d6734
+name: SVG image has accessible name
+rule_type: atomic
+description: |
+   This rule checks that each SVG image element that is included in the accessibility tree has an accessible name.
+accessibility_requirements:
+  wcag20:1.1.1: # Non-Text Content (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+authors:
+  - Bryn Anderson
+---
+
+## Applicability
+
+The rule applies to any element in the [SVG](https://www.w3.org/2000/svg) namespace with a [semantic role](#semantic-role) of either `img`, `graphics-document`, `graphics-object`, `graphics-symbol`, that is [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
That line should be removed (duplicated with more details (visibility) just below).